data/mods/Magiclysm/professions.json

```diff
@@ -18,7 +18,7 @@
     "description": "You have loved fire all of your life.  You have now discovered your inner fire, and want to exercise that upon the world.",
     "points": 2,
     "items": {
-      "both": [ "lighter", "sneakers", "pants", "tshirt", "jacket_light", "pyro" ],
+      "both": [ "lighter", "sneakers", "pants", "tshirt", "jacket_light", "pyro", "mbag" ],
```

Comment:
This is the only one I saw an issue with, the other professions were able to hold everything without being forced to wield an item.